public/js/models/list.js

```diff
@@ -0,0 +1,20 @@
+var List = function(info){
+  var self = this;
+  this.title = info.title;
+}
+
+List.fetch = function(){
+  var request = $.getJSON("http://localhost:3000/lists")
```

Comment:
If you remove the protocol, host, and port, this application will be easier to deploy.